.github/workflows/push_ocm.yaml

```diff
@@ -70,3 +67,67 @@ jobs:
             ghcr.io/open-component-model/ocm/ocm.software/ocmcli/ocmcli-image:latest
             ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+  ocm-cli-latest:
+    name: ocm-cli build latest
+    runs-on: large_runner
+    permissions:
+      contents: write
+      id-token: write
+      packages: write
+    steps:
+    - name: Self Hosted Runner Post Job Cleanup Action
+      uses: TooMuch4U/actions-clean@v2.2
+    - name: Generate token
+      id: generate_token
+      uses: tibdex/github-app-token@v2
+      with:
+        app_id: ${{ secrets.OCMBOT_APP_ID }}
+        private_key: ${{ secrets.OCMBOT_PRIV_KEY }}
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        token: ${{ steps.generate_token.outputs.token }}
+    - name: Enable gh-CLI
+      uses: actions4gh/setup-gh@v1.0.2
+      with:
+        token: ${{ steps.generate_token.outputs.token }}
+    - name: delete release and move tag
+      run: |
+        if [ "$(gh api -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" /repos/open-component-model/ocm/releases | grep -c 'ocm-cli/latest')" -ge 1 ]; then
+          RELEASE_ID=$(gh api -X GET -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" /repos/open-component-model/ocm/releases/tags/ocm-cli/latest | jq -r .id)
+          gh api -X DELETE -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" /repos/open-component-model/ocm/releases/$RELEASE_ID
+        fi
+        gh api -X PATCH -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" /repos/open-component-model/ocm/git/refs/tags/ocm-cli/latest -f "sha=${{ github.sha }}" -F "force=true"
+    - name: Setup Go
+      uses: actions/setup-go@v5
+      with:
+        go-version-file: '${{ github.workspace }}/go.mod'
+        check-latest: false
+    - name: goreleaser release snapshot
+      uses: goreleaser/goreleaser-action@v6
+      with:
+        distribution: goreleaser
+        version: 2.2.0
+        args: release --clean --timeout 60m --snapshot --skip=validate --config=.github/config/latest.yml
+      env:
+        GITHUBORG: ${{ github.repository_owner }}
+        GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
+        GORELEASER_CURRENT_TAG: v0.0.0
+        GORELEASER_PREVIOUS_TAG: v0.0.0
+    - name: remove some meta files
+      run: |
+        rm -rf dist/*.json dist/*.yaml
+    - name: Create Release
+      uses: Hs1r1us/Release-AIO@v2.0.0
```

Comment:
This action does not seem to be maintained actively. The last commit was two years ago and there is still a TODO open (see their README.md).

Maybe this action could be more suitable? https://github.com/ncipollo/release-action/tree/main

I did not look too deep into it, but it seems that it also provides a functionality to make `latest` releases and replace artifacts.